index.html

```diff
@@ -4,9 +4,39 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <link rel="stylesheet" href="/style.css">
+    <title>Mobile Portfolio</title>
 </head>
 <body>
-    
+    <div class="container">
+        <header>
+            <div class="flex-container">
+                <div class="flex-items"><nav><img src="images/menu.png" alt="menu"> </nav></div>
+            </div>
+        </header>  
+        <section class="headline">
+            <h1>Charlie<br>Giorgania</h1>
+            <p>Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works.
+            If you like what you see and have a project you need coded, don't hesitate and contact me.</p> 
+            <button type="button" class="rectangle">Start collaboration</button>             
+        </section>
+    </div>
+    <div><img src="images/geometry.png" alt="menu"></div>
```

Comment:
- [ ] this `img` tag should be removed because, decorations are always background images, not `img` tags. please include this.